plugin/evm/config.go

```diff
@@ -150,10 +153,13 @@ type Config struct {
 	KeystoreInsecureUnlockAllowed bool   `json:"keystore-insecure-unlock-allowed"`
 
 	// Gossip Settings
-	PushGossipFrequency Duration `json:"push-gossip-frequency"`
-	PullGossipFrequency Duration `json:"pull-gossip-frequency"`
-	RegossipFrequency   Duration `json:"regossip-frequency"`
-	TxRegossipFrequency Duration `json:"tx-regossip-frequency"` // Deprecated: use RegossipFrequency instead
+	PushGossipNumValidators    int      `json:"push-gossip-num-validators"`
+	PushGossipNumNonValidators int      `json:"push-gossip-num-non-validators"`
+	PushGossipNumPeers         int      `json:"push-gossip-num-peers"`
+	PushGossipFrequency        Duration `json:"push-gossip-frequency"`
+	PullGossipFrequency        Duration `json:"pull-gossip-frequency"`
+	RegossipFrequency          Duration `json:"regossip-frequency"`
```

Comment:
nit: would change the var name to `PushRegossipFreqency` and put above `PullGossipFrequency`

Might make sense to deprecate `regossip-frequency` too but we can probably leave all of this for future work.

Reply:
Yeah I agree that would be more clean - but I think that would make sense to do in its own PR. Seems unrelated to this change